Explicitly denying on-prem connectivity (#3233)
astronomer/docs: astro/connect-aws.md

- Transit Gateways
- AWS PrivateLink

> If you require a private connection between Astro and AWS, Astronomer recommends configuring a dedicated cluster. See [Create a dedicated cluster](create-dedicated-cluster.md). Transitive connectivity to on-premise networks is also possible through customer managed VPCs; architectures with a demarcation point between Astro and the customers on-premise network are not supported.

Reviewer: ```suggestion
If you require a private connection between Astro and AWS, Astronomer recommends configuring a dedicated cluster. See [Create a dedicated cluster](create-dedicated-cluster.md). Transitive connectivity to on-premise networks is also possible through your managed VPCs. However, architectures with a demarcation point between Astro and your on-premise network are not supported.
```
